### PR TITLE
Ukrainian translation for OCS 2.3.

### PR DESCRIPTION
**Pull request by KuznetsovIllya**
Ukrainian translation for OCS 2.3.

### CODE REVIEW

File: locale/uk_UA/admin.xml [hunk 1]
```diff
@@ -0,0 +1,68 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE locale SYSTEM "../../lib/pkp/dtd/locale.dtd">
+
+<!--
+  * ocs2-devel/locale/en_US/admin.xml
```
Comment (mtub) [inline locale/uk_UA/admin.xml]: path should be changed to ocs2-devel/locale/uk_UA/admin.xml

File: locale/uk_UA/admin.xml [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+  *
+  * Copyright (c) 2000-2012 John Willinsky
+  * Distributed under the GNU GPL v2. For full terms see the file docs/COPYING.
+  *
+  * Localization strings for the en_US (English) locale.
```
Comment (mtub) [inline locale/uk_UA/admin.xml]: ... for the uk_UA (Ukrainian (Ukraine)) locale.

File: locale/uk_UA/admin.xml [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+  *
+  * $Id$
+  -->
+
+<locale name="en_US" full_name = "English">
```
Comment (mtub) [inline locale/uk_UA/admin.xml]: <locale name="uk_UA" full_name = "Ukrainian (Ukraine)">

**Comment by mtub** [comment]
Thanks, @KuznetsovIllya, this looks good. I will merge the commits and then make the header changes I've suggested after that. When adapting files from the English version, you should make sure to edit the locale name (en_US -> uk_UA), too.

**Comment by KuznetsovIllya** [comment]
Nice, thanks, @mtub. I've used the translation plugin - that's why I didn't know, that headers should be changed.

**Pull request merged**